spec/rubocop/cop/layout/tab_spec.rb

```diff
@@ -22,9 +22,9 @@
   end
 
   it 'registers an offense for a line indented with mixed whitespace' do
-    expect_offense(<<-'RUBY'.strip_indent)
-       	x = 0
-       ^ Tab detected.
+    expect_offense(<<~'RUBY')
+      	 x = 0
```

Comment:
I changed the test code. It was `[space][tab]x = 0`, but it is meaningless. Because the `[space]` is removed by `strip_indent`.
So it is changed to `[tab][space]x = 0` to avoid removing the space.